### PR TITLE
[ODS-5737] Clean up obsolete Profiles-related artifacts from solution

### PR DESCRIPTION
**Pull request by gmcelhanon**
(This builds on the other PR opened by Semalai but is a deeper cleaning)

### CODE REVIEW

File: Application/EdFi.Ods.Common/Conventions/ExtensionsConventions.cs [hunk 1]
```diff
@@ -77,34 +74,13 @@ public static string GetExtensionClassAssemblyQualifiedName(Type edFiStandardTyp
             {
                 //If standard type exists in profile assembly, use edFiStandardType assembly name.
```
Comment (simpat-jesus) [inline Application/EdFi.Ods.Common/Conventions/ExtensionsConventions.cs:75]: Is this comment still relevant to be updated or can be removed?

**Review by simpat-jesus — CHANGES_REQUESTED** [review]
Looks good, there is only one minor comment, I can approve and merge when it is resolved

File: Application/EdFi.Ods.Common/Conventions/ExtensionsConventions.cs (hunk 1 above)
Comment (gmcelhanon) [reply Application/EdFi.Ods.Common/Conventions/ExtensionsConventions.cs]: I cleaned this up.

**Review by simpat-jesus — APPROVED** [review]

**Comment by simpat-jesus** [comment]
TC builds check fails are unrelated to this, can be ignored for now since the same tests are passing in Github actions
